Publish initial draft of the emnify Portal user guide (#142)
emnify/product-docs: docs/portal/sims-and-devices/connected-devices.md

---
description: Learn how to manage your devices via the Connected Devices page in the emnify Portal
slug: /portal/connected-devices
---

# Connected Devices

> You can manage your devices from the [**Connected Devices**](https://portal.emnify.com/connected-devices) page.

Reviewer: ```suggestion
 [**Connected Devices**](https://portal.emnify.com/connected-devices) is where you manage your devices.
```

<details className="custom-details-example">
  <summary>Show me an example of Device Details</summary>
  <img
    src={require('./assets/device-details.png').default}
    alt=""
  />
</details>

### IMSI

The international mobile subscriber identity (IMSI) is a unique number used to identify a GSM subscriber. 
Therefore it will change if a device connects to a different operator while roaming.

<details className="custom-details-tip">
  <summary>See how the IMSI is constructed</summary>
  An IMSI is usually a 15-digit number but can be 14 digits in some cases.   
  
  It comprises three components: MCC, MNC, and MSIN.

  MCC: 3-digit country identifier  
  MNC: Operator identifier (2 or 3 digits)  
  MSIN: Identifier for the connected device (9 or 10 digits)  

  <img
    src={require('./assets/imsi-components-table.png').default}
    style={{width:300}}
    alt=""
  />
</details> 

### MSISDN

The mobile station international subscriber directory number (MSISDN) is essentially the telephone number assigned to each SIM.
For an emnify IoT eSIM, this number is needed for its ability to receive SMS messages (i.e., [SMS MT](/glossary#sms-mt)).

> ### IMEI

Reviewer: This is a lot of glossary information. Is there any way to feed the data here from the glossary so there is one location to update it?

Author: Would something like [what I did for the workspaces page](https://github.com/emnify/product-docs/pull/142/files#diff-7ba7fd7ae40ea47be164a2e3acf83f864d1c667cc84d6837084e62a77f34f711R14-R20) work? Then I can update the Glossary entries with any new data presented here.

The international mobile equipment identity (IMEI) is a unique number used to identify cellular modems.

<details className="custom-details-tip">
  <summary>Learn more about IMEIs</summary>
  An IMEI has 15 digits (14 digits plus a check digit).

  An IMEISV has 16 digits (14 digits plus two software version digits).

  For more detailed information, see emnify's IoT Glossary article: [What is an IMEI number?](https://www.emnify.com/iot-glossary/imei-number)
</details>

## Device status

A device has only two states: **Enabled** or **Disabled**.

You can toggle one or more devices between these states to control their data usage.
A disabled device doesn't incur any charges.

> When you are ready to deploy a device and want it to connect to a network, change its status to **Enabled**.

Reviewer: ```suggestion
When you want a device to connect to a network, change its status to **Enabled**.
```

> To change a device's status, use the **Status** toggle from the **Connected Devices** list or the device's **Details** page.

Reviewer: ```suggestion
To do this, use the **Status** toggle from the **Connected Devices** list or the device's **Details** page.
```
If the content is clearly connected with the previous sentence.

Were a lot of line breaks added to make the content stay on the left side of the page? If feels like we have a sentence per line...

Author: I broke it up so it's easier to skim and find what you need, but I can have another look to make sure it makes sense in the context 👀

You can also perform a bulk Enable/Disable operation on a group of devices.

> Use a filter to display a list of the targeted devices.

Reviewer: This doesn't fit well together. The formatting makes it unclear that this paragraph is a "how-to"
